hikari/messages.py

```diff
@@ -589,6 +601,15 @@ class ButtonStyle(int, enums.Enum):
 """
 
 
+class TextInputStyle(int, enums.Enum):
```

Comment:
```suggestion
class ModalTextInputStyle(int, enums.Enum):
```

Reply:
I think using `TextInputStyle` would me more consistent as `ButtonStyle` isnt `MessageButtonStyle` or something like that.
[This too](https://discord.com/channels/574921006817476608/644130410564943873/948896650380849152)

Reply:
Unfortunately, I have decided to decline this suggestion.